config/src/config/logger_config.rs

```diff
@@ -13,40 +12,15 @@ pub struct LoggerConfig {
     // chan_size of slog async drain for node logging.
     pub chan_size: usize,
     // The default logging level for slog.
-    #[serde(serialize_with = "serialize_filter_level")]
-    #[serde(deserialize_with = "deserialize_filter_level")]
-    pub filter_level: FilterLevel,
+    pub level: Level,
 }
 
 impl Default for LoggerConfig {
     fn default() -> LoggerConfig {
         LoggerConfig {
             is_async: true,
             chan_size: 256,
```

Comment:
chan_size is defaulted to 256 here and 128 in the logger code. Is there a reason to expose this parameter at all? I would recommend we just delete it from LoggerConfig and set the 256 or 128 explicitly in the logger init or wherever. And then delete chan_size from all the config files.

If this needs to be configured, then I'd add a const to the logger for the default chan size, and import and use it here.

Reply:
Good call, with no intent to rock the boat too much, let's add the const and figure this one out later...